fabrics.c

```diff
@@ -650,6 +652,18 @@ static int discover_from_json_config_file(nvme_root_t r, nvme_host_t h,
 			    strcmp(transport, "fc"))
 				continue;
 
+			/* ignore if no host_traddr for fc */
+			if (!strcmp(transport, "fc")) {
+				if (!host_traddr)
+					continue;
```

Comment:
Should we print an error message to `stderr` indicating that there is an error in the JSON file?